{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/cli.py

```diff
@@ -25,6 +25,7 @@
 
 from .app_logging import configure_logging_to_terminal
 from .make_specs import build_model_specifications
+from .make_artifacts import build_artifacts
```

Comment:
I think I got lazy when I wrote these as relative imports.  I typically have a strong preference for absolute imports.  They're more clear and less brittle and they work better with IDE tooling.  An option: in the tools module init, import the application main functions, then just `from project import tools` here and make dotted calls to the application functions (e.g. tools.build_model_specifications).

Reply:
Changed.